src/main/java/core/basesyntax/service/operations/ReturnOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction) {
+        storageFruits.put(transaction.getFruit(), storageFruits
+                .getOrDefault(transaction.getFruit(), 0) + transaction.getQuantity());
```

Comment:
It appears that 'storageFruits' is being used without being defined in this class. Ensure that all necessary dependencies are properly injected into the class. If 'storageFruits' is a part of the 'Storage' class, it should be accessed through an instance of 'Storage' or through a service that encapsulates the storage logic.